### ISSUE TITLE
Stara Impreador 3.0 Multiple Odd Actions

### ISSUE THREAD

**Issue by lcc1211** [Courseplay/courseplay#2731]
I searched for this because I thought someone would have reported this by now, but I didn't find it.  The sprayer/spreader combo has the following issues:
1. At start of course it spreads the sprayer boom and then closes it before entering the field. 
2. On the field it only spreads the solid fertilizer. 
3. Once the solid fertilizer is gone, it doesn't switch to the liquid, but rather spreads the sprayer boom and heads over to the refill point.
Standard Goldcrest map, all fields.  Other mod maps as well. Latest (as of posting) version of DEV CP.
Link to the video:  https://gaming.youtube.com/watch?v=aqp8dyeSX40&feature=share
Log:
[log.txt](https://github.com/Courseplay/courseplay/files/2020574/log.txt)
Sorry for the very zoomed in video, but I had to do this from my laptop which has no mouse and I couldn't figure out how to zoom out, but I think the video shows enough.

Just a minor update. If you manually select the liquid fertilizer, CP won't even start the vehicle moving. It sits on the field taking no action.
Ideally, it would be fantastic if CP could recognize that there are 2 forms of fertilizer to fill much like it recognizes a sower with both fertilizer and seeds to fill.  Further, it would be great if CP would be able to use both the solid and the liquid, but one at a time.

**Comment by MoslessNeo** [comment]
This is not a bug. Just one detail.
Leave the bars open before starting work. (at the same time, see in the help box what is the key to switch from solid fertilizer to liquid).

**Comment by lcc1211** [comment]
I think the bug is the bars open at the end of the course and on most maps this interferes with the route I create to the refill point as they are 36 meters wide.

**Comment by pops64** [comment]
CP was never designed to handle two types of fertilizer at the same in mode 4. This probably will be a won't fix
